docs/api/topics/latecustomization.rst

```diff
@@ -0,0 +1,65 @@
+.. _latecustomization:
+.. versionadded:: 2
+
+====================
+ Late Customization
+====================
+
+Late customization allows loading of apps during the first-time experience on
+devices.
+
+.. http:get:: /api/v2/latecustomization/?carrier=(str:carrier)&region=(str:region)
+
+  Returns a list of late customization items for this carrier/region.
+
+  **Response**
+
+  :param app: A webapp slug, if item is a webapp.
+  :param extension: An addon slug, if item is an addon.
+  :param url: URL for package minifest.
+  :status 200: Successfully completed.
+
+
+  .. code-block:: json
+
+      {
+          "objects": [
+              {
+                  "app": "carrier-provided-app-1",
+                  "url": "https://marketplace.firefox.com/app/21066c1c-e45d-4538-8f80-96981df51468/manifest.webapp"
```

Comment:
It would probably be useful to include the type of the object as well. `"type": "app"` or something.